modules/services/activate-system/default.nix

```diff
@@ -42,5 +42,10 @@ in
       serviceConfig.KeepAlive.SuccessfulExit = false;
     };
 
+    launchd.user.agents.activate-user = {
+      command = "/run/current-system/activate-user";
+      serviceConfig.RunAtLoad = true;
+      serviceConfig.KeepAlive.SuccessfulExit = false;
+    };
```

Comment:
This will run `activate-user` as every user on a machine which is not ideal

Reply:
Thanks for this comment, I didn't realize it.

On a related note, I am now realizing that there's also `launchd.agents` module in `home-manager`. Is it a different module? Does it mean I can achieve the same effect (scoped to my user) by defining it there instead? If so, then I think I may not need this patch in `nix-darwin` at all. I can instead define the service that `launchctl setenv` in HM instead. (That's the only part of the `activate-user` script I actually care about here.)

Reply:
Yeah, putting it in home-manager will definitely work better as it’ll be specific to your user